analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyCollectionsShouldNotBeEnumerated.CSharp9.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+
+var topLevel = new List<int>();
+topLevel.Clear();   // FN
+
+topLevel.Add(42);
+topLevel.Clear();
+
+void TopLevelLocalFunction()
+{
+    var list = new List<int>();
+    list.Clear();   // FN
+
+    list.Add(42);
+    list.Clear();
+}
+
+public class Tests
+{
+    public void TargetTypedNew()
+    {
+        List<int> list;
+
+        list = new();
+        list.Clear();   // FN
+        list.Add(42);
+        list.Clear();
+
+        list = new(42);
+        list.Clear();   // FN
+        list.Add(42);
+        list.Clear();
+
+        list = new(new[] { 42 });
+        list.Clear();   // Compliant
+    }
```

Comment:
There was a test case removed from here. Only the comment should have been replaced with FN
```
        list = new List<int>();
        list.Clear();   // FN, can't build CFG for this method
```